project-koku/koku
# Remove AWS CCSP data from HCS Filter

## Jira Ticket

[COST-4366](https://issues.redhat.com/browse/COST-4366)

## Description

This change will remove the CCSP data filter from HCS filtering for AWS leaving only the private offer data.

## Testing

1. Checkout Branch
2. Restart Koku
3. make create-test-customer
4. make load-test-customer-data test_source=AWS
5. Check minio for the HCS AWS data and ensure all data is for a private AWS marketplace offer with a legalentity of Red Hat

## Notes

...
